jekyll/_cci2/orbs-faq.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: classic-docs
+title: "Orbs FAQ"
+short-title: "Orbs FAQ"
+description: "FAQs for Orbs"
+categories: [configuring-jobs]
+order: 20
+---
+
+This document describes various questions and technical issues that you may find helpful when working with Orbs.
+
+* TOC
+{:toc}
+
+## Introduction
+{:.no_toc}
+
+When using Orbs or authoring your own Orbs, you may come across some issues that may not be directly addressed in the documentation. This section attempts to address specific issues and questions that have been relayed to CircleCI during the initial design and development phases with internal and external customers. Because Orbs development and design will be a continuous process, this section will be frequently updated as needed to ensure you the latest technical information is available.
+
+### Downloading, Integrating, and Testing Orbs
+
+* Question: How do I download, integrate and test Orbs?
+
+* Answer: You can invoke Orbs using the `orbs` stanza in version 2.1 or higher of a CircleCI configuration. For example, if you want to publish an Orb called  `hello-build` in the namespace `circleci` and have a published version `0.0.1`, invoke an orb like the example shown below:
+
+```orbs:
+hello: circleci/hello-build@0.0.1
+```
+
+You may then use the elements of that orb in your configuration by referencing these elements under the key `hello`. For example, if an orb has a job called `hello-build`, you can invoke that job in a workflow like the example shown below.
+
+```workflows:
+  info:
+    jobs:
+      - hello/hello-build
+```
+
+CircleCI will publish a web-based registry viewer in the future so Orbs documentation can be auto-generated. In the meantime, you can always pull the source of Orbs directly. For example, you can run `circleci orb source circleci/hello-build@0.01`
+
+### Build Error When Testing Locally
+
+* Question: Why do I get the following error when testing locally:
+
+```circleci build -c .circleci/jobs.yml --job test
+```
+
+```Error:
+You attempted to run a local build with version 2.1 of configuration.
+```
+
+* Answer: To resolve this error, run `circleci config process` on your configuration and then save that configuration to disk. You then should run `circleci build` (this has now been moved to `circleci local execute`) against the processed configuration.
```

Comment:
We should go ahead and stop documenting `circleci build` in favor of `circleci local execute` across the board.

Reply:
thanks, fixed